scripts/set-performance

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+source env-prep
+PERSIST=${PERSIST:-"transient"}
```

Comment:
Should the default really be `transient`?  If they are improving performance wouldn't we want them to be persistent?